web/webkit/src/main/resources/toserve/lift.js

```diff
@@ -397,6 +400,37 @@
       }
     }
 
+    function rehydrateComets() {
+      function onSuccess(html) {
+        var iframe = document.createElement("iframe");
+        var doc = document.createElement("html");
+        iframe.appendChild(doc);
+        doc.innerHTML = html;
+
+        toWatch = {};
```

Comment:
Is this attached to window or a lift object on the window?

Reply:
It's declared up at the top at [line 20](https://github.com/lift/framework/blob/4c1e2ad004282b12f38db6dae1b954e508f4506a/web/webkit/src/main/resources/toserve/lift.js#L20)